### ISSUE TITLE
Add indicator of Actionable Items in Task Thread Header

### ISSUE THREAD

**Issue by jingconan** [hbhzwj/gtd-gdocs#24]

**Comment by jingconan** [comment]
If we are doing a task, we realize that we need to finish some other tasks before going back to this task. Then the new task will be the subtask of this task. 

Subtasks define a dependency relationship. Subtask must be finished before the father task can be finished. 

In the task header, I should show # of subtasks and give the link

**Comment by jingconan** [comment]
I should focus on simplifying workflow. Subtasks make things too complicated, I don't think it is very useful.

**Comment by jingconan** [comment]
Although there is no need to show the # of subtasks. It will be very useful to identity the number of Actionable Items (AIs) in a task thread header.

**Comment by jingconan** [comment]
In addition to adding a indicator, I should also make status of comment changeable.

Make the prefix of the first paragraph of comment change from these status:
- AI: 
- WF: 
- Done: 

I can reuse the 'move to actionable', 'move to waiting for', 'move to done' submenus in GTD menu.

When a user clicks 'move to waiting for' for a task thread, show a popup saying: 'what are you waiting for?'. After getting user's input, create a new comment with the input text as the first paragraph.

Similarly, when a user clicks 'move to actionable', we can ask 'what is the next Actionable Item?'

When user click 'done', mark all actionable items done.
